docs/models.md

```diff
@@ -1,7 +1,7 @@
 
 ## ImageData
 
-Reader methods returning image data  (`tile`, `part` and `preview`) return a data holding class (built with [attrs](https://www.attrs.org/en/stable/)): `rio_tiler.models.ImageData`.
+Reader methods returning image data  (`tile`, `part` and `preview`) return a *data holding* class (built with [attrs](https://www.attrs.org/en/stable/)): `rio_tiler.models.ImageData`.
```

Comment:
Nit: I'd remove the "built with attrs" because that isn't information the _user_ cares about